### DISCUSSION TITLE
Problem with sending a report via trcli using --suite-id parameter

### DISCUSSION THREAD

**Discussion by merloty** [gurock/trcli#252] [Q&A]
On a previous version of **TestRail** I sent reports using the _f'trcli -y -h request {URL} --project "{PROJECT_NAME}" --username {TESTRAIL_USERNAME} --password {TESTRAIL_PASSWORD} parse_junit --title "{devname}_TestRun" -f {juitxml} --case-fields custom_minimal_version:7 --case-fields custom_case_type_combo:[5] --case-fields custom_supporteddevice:[0] --case-fields custom_security_test:1 --milestone-id {milestone_id} --suite-id 10723' _

After upgrading to version **v8.0.4** there comes some problem with parameters, I get this text: _**Error occurred while checking for 'missing sections': 'Field :limit is too small (minimum 1).'**_ From the error it is not clear what parameter we are talking about and what is wrong with it. Experimentally found out that the problem occurs with **--suite-id**, because if you remove it, the error disappears.

Suite exists, because:
_/index.php?/api/v2/get_suite/10723 returns:_
_{_
_ "id": 10723,_
_ "name": "Automated Smoke Tests",_
_ "description": null,_
_ "project_id": 4,_
_ "is_master": false,_
_ "is_baseline": false,_
_ "is_completed": false,_
_ "completed_on": null,_

_}_

**Comment by acuanico-tr-galt** [comment]
Hi, based on the error there seems to have an issue with your fields. You may enable verbose logging using -v option to know detailed info about the issue (if you are still encountering it on the latest version). Please raise an issue with a sample results file and steps to replicate if this is still a problem. Thank you.
